src/OpenTK/Platform/Windows/WinRawKeyboard.cs

```diff
@@ -304,5 +307,11 @@ public string GetDeviceName(int index)
                 }
             }
         }
+        
+        private void UpdateModifiers(ref KeyboardState state)
+        {
+            state.CapsLock = (Functions.GetKeyState(VirtualKeys.CAPITAL) & 1) == 1;
+            state.NumLock = (Functions.GetKeyState(VirtualKeys.NUMLOCK) & 1) == 1;
+        }
```

Comment:
This should probably be done inside ProcessKeyboardEvent rather than in GetState